asciidoc-confluence-publisher-docker/Dockerfile

```diff
@@ -19,18 +21,4 @@ ENV SOURCE_ENCODING="" \
     VERSION_MESSAGE="" \
     ATTRIBUTES=""
 
-ENTRYPOINT ["sh", "-c", "java -jar /opt/asciidoc-confluence-publisher-docker.jar \
-    \"asciidocRootFolder=/var/asciidoc-root-folder\" \
-    \"sourceEncoding=$SOURCE_ENCODING\" \
-    \"rootConfluenceUrl=$ROOT_CONFLUENCE_URL\" \
-    \"skipSslVerification=$SKIP_SSL_VERIFICATION\" \
-    \"spaceKey=$SPACE_KEY\" \
-    \"ancestorId=$ANCESTOR_ID\" \
-    \"username=$USERNAME\" \
-    \"password=$PASSWORD\" \
-    \"pageTitlePrefix=$PAGE_TITLE_PREFIX\" \
-    \"pageTitleSuffix=$PAGE_TITLE_SUFFIX\" \
-    \"publishingStrategy=$PUBLISHING_STRATEGY\" \
-    \"versionMessage=$VERSION_MESSAGE\" \
-    \"attributes=$ATTRIBUTES\" \
-"]
+ENTRYPOINT ["sh", "-c", "/opt/asciidoc-confluence-publisher.sh"]
```

Comment:
Even slicker would be to just have a single helper script listed in the `ENTRYPOINT`, e.g. similar to how PostgreSQL is doing it (see https://github.com/docker-library/postgres/blob/cf9b6cdd64f8a81b1abf9e487886f47e4971abe2/11/Dockerfile, just without any `CMD`), maybe called `publish.sh` or `docker-entrypoint.sh`. 

In addition, we should consider adding a paragraph to the documentation on the use of the Docker image in the context of build systems (i.e. your scenario) and document there the name of the entrypoint helper script (and thus making it public API). If you agree, would you mind extending your pull request accordingly? Thank you!

Reply:
Feel free to extend this MR as far as needed ;-)

Because the script may be used by someone I decided to use this rather long name but maybe you're right and `publish.sh` would be fine, too.

After some thoughts I'm not sure if using `ENTRYPOINT` may be the best option. If this is used, instead of `CMD`, in our case we need to override the `ENTRYPOINT` to `[""]` to be able to defined GitLab CI scripts. But if you insist on keeping the `ENTRYPOINT` this is fine, too. Using the script directly without `"sh", "-c"` should be fine though.

Reply:
I added 5018b7d for renaming the script and 390ed86 which adds a small section, including an example, to the documentation.